linera-views/src/dynamo_db.rs

```diff
@@ -104,8 +104,14 @@ pub const TEST_DYNAMO_DB_MAX_STREAM_QUERIES: usize = 10;
 
 /// Fundamental constants in DynamoDB: The maximum size of a TransactWriteItem is 100.
 /// See <https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_TransactWriteItems.html>
+#[cfg(not(all(feature = "test", target_os = "macos")))]
 pub const MAX_TRANSACT_WRITE_ITEM_SIZE: usize = 100;
 
+/// The maximum size of a TransactWriteItem appears to be still 25 for dynamodb-local on
+/// MacOS.
```

Comment:
I saw that on Linux, too, and I just had to update my Docker container. Does that really have something to do with MacOS and not just the DynamoDB version?